packages/agreement-process/src/utilities/errorMappers.ts

```diff
@@ -124,3 +124,12 @@ export const suspendAgreementErrorMapper = (
     .with("operationNotAllowed", () => HTTP_STATUS_FORBIDDEN)
     .with("agreementNotInExpectedState", () => HTTP_STATUS_BAD_REQUEST)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const rejectAgreementErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("agreementNotFound", () => 404)
+    .with("agreementNotInExpectedState", () => 400)
+    .with("operationNotAllowed", () => 403)
+    .otherwise(() => 500);
```

Comment:
now are using HTTP status const 
```suggestion
  match(error.code)
    .with("agreementNotFound", () => HTTP_STATUS_NOT_FOUND)
    .with("agreementNotInExpectedState", () => HTTP_STATUS_BAD_REQUEST)
    .with("operationNotAllowed", () => HTTP_STATUS_FORBIDDEN)
    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
```